### PR TITLE
Improve Postfinance FDS modules Usability

### PR DESCRIPTION
**Pull request by ecino**
This pull request makes usability improvements for working with Postfinance FDS files. It adds a parser for the statement imports, resolves some bugs and change the views.
### l10n_ch_account_statement_base_import
- Add a XML Camt parser for Postfinance statements. This is the new standard format used by Postfinance.
- Some archives contains .tiff attachments that must be converted to png in order to be shown in Odoo
### l10n_ch_fds_postfinance
- Fix some remaining bugs (wrong class names)
- Change the views to make it easier to import files
- Add exclusion possibility to avoid downloading all files from the SFTP

PS: Pull Request #213 should be aware of these changes.

### CODE REVIEW

**Comment by ecino** [comment]
Travis error comes from l10n_ch_dta/demo/dta_demo.xml:10 invalid BIC bank format, not related to this PR.

**Comment by ecino** [comment]
Travis error is still from dta module, not related to this PR:

ParseError: "null value in column "type" violates not-null constraint

" while parsing /home/travis/build/OCA/l10n-switzerland/l10n_ch_dta/demo/dta_demo.xml:37, near

<record model="payment.mode" id="payment_mode_dta">

```
<field name="name">DTA</field>

<field name="bank_id" ref="main_partner_bank"/>

<field name="journal" model="account.journal" search="[('type','=','cash')]"/>
```

  </record>

File: l10n_ch_account_statement_base_import/models/account_bank_statement_import.py [hunk 1]
```diff
@@ -181,3 +181,18 @@ def _create_bank_statement(self, stmt_vals):
...
+
+    @api.model
+    def _find_bank_account_id(self, account_number):
+        """ Override to find Postfinance account Given IBAN number """
+        bank_account_id = super(AccountBankStatementImport, self).\
+            _find_bank_account_id(account_number)
+        if not bank_account_id and account_number and len(
+                account_number) == 21:
+            pf_formated_acc_number = account_number[9:].lstrip('0')
+            bank_account_ids = self.env['res.partner.bank'].search(
```
Comment (yvaucher) [inline l10n_ch_account_statement_base_import/models/account_bank_statement_import.py]: Please rename this variable `bank_account` you don't have an id but a record set here. Plus this record set only contains one element.

File: l10n_ch_account_statement_base_import/models/account_bank_statement_import.py [hunk 2]
```diff
@@ -181,3 +181,18 @@ def _create_bank_statement(self, stmt_vals):
...
+                [('sanitized_acc_number', '=', pf_formated_acc_number)],
+                limit=1)
+            bank_account_id = bank_account_ids[0].id if bank_account_ids\
```
Comment (yvaucher) [inline l10n_ch_account_statement_base_import/models/account_bank_statement_import.py]: Simply do `bank_account_id = bank_account.id`

If is a record set `rec` has 0 elements, `rec.id` returns `False`

**Comment by yvaucher** [comment]
Just few remarks otherwise LGTM

**Comment by ecino** [comment]
Corrected and squashed some commits.

**Comment by yvaucher** [comment]
:+1:  Thanks

**Pull request merged**